frontend/src/shared/components/create-organization/create-organization-dialog/create-organization-dialog.component.html

```diff
@@ -0,0 +1,26 @@
+<h1 mat-dialog-title>Create organization</h1>
+    <div class="create-org-dialog">
+    <mat-dialog-content class="dialog-content">
+        <span class="input-title">Name</span>
+        <input 
+            class="organization-name-input" 
+            type="text"
+            id="organizationName"
+            #organizationName
+        >
```

Comment:
Use our tasque-input component

Reply:
This one was made before this input was created